examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -163,7 +187,8 @@ List<GalleryDemo> _buildGalleryDemos() {
       icon: GalleryIcons.bottom_sheets,
       category: _kMaterialComponents,
       routeName: ModalBottomSheetDemo.routeName,
-      documentationUrl: 'https://docs.flutter.io/flutter/material/showModalBottomSheet.html',
+      documentationUrl:
```

Comment:
Please don't reformat existing code (here and elsewhere)

Reply:
@HansMuller The file is formatted using `dartfmt` I can selectively go back and unformat those lines but do you have a better solution?

Reply:
We don't use dartfmt. The code was largely formatted by hand per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

It looks like this file was the only one that got wholesale reformatted.

Reply:
I reverted that file